scripts/index.js

```diff
@@ -51,3 +51,6 @@ try {
 const weatherDiv = document.querySelector('[data-weather]');
 
 
+const cityName = document.createElement('p');
+cityName.textContent = `City: ${getLocationName(atlWeather)}`;
```

Comment:
Good use of string interpolation! I like that you skipped the intermediary variable for this one.